src/space/encounter.js

```diff
@@ -240,6 +135,15 @@ export default class Encounter {
         this.markedForInsertion = [];
     }
 
+    win() {
+        // TODO render stuff
+        console.log("You have just won")
+    }
+    loose() {
```

Comment:
Just a typo, but I think you meant _lose_ as to not win and not _loose_ as in not tight ;)

Reply:
Yes, thats what I meant